frontend/vue/components/CodeExercise/CodeOutput.vue

```diff
@@ -102,6 +103,7 @@ export default defineComponent({
       })
     },
     _executeCode (code: string) {
+
```

Comment:
if we remove the extra blank lines in this file then we would not need to commit any changes to the file.